docs/platform/howto/access-suspended-due.rst-to-past-due-invoice

```diff
@@ -0,0 +1,24 @@
+Access Suspended Due to Past Due Invoice
+===================================================================
+
```

Comment:
```suggestion

If you have bills past due and didn't set up a payment method, you will receive this error when trying to log in to the console.

ERROR
Project suspended, access prohibited

```

Reply:
I think we can add this here as an introduction to the article. :)

Reply:
There's a special block for warnings and errors: see docs/platform/howto/payment-issues-plan-upgrades.rst for an example